community-resources/resources/META-INF/SherlockPlatformPlugin.xml

```diff
@@ -0,0 +1,33 @@
+<idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude">
+  <!-- TODO: can the Java plugin be excluded somehow? -->
+  <xi:include href="/META-INF/JavaIdePlugin.xml" xpointer="xpointer(/idea-plugin/*)"/>
+
+  <!--
```

Comment:
We shouldn't be needing this block. Do you get runtime errors without this? (This is related to the v2 modules in the platform.) Did you get this from GameTools (we solved it a different way in game tools)?

Reply:
I tried after patching this PR, and it looks like the platform build fails without SettingsControlle.  

Please try replacing this block with this:

```
  <xi:include href="/META-INF/essential-modules.xml"/>
```

The following can be even leaner, but might be less future-proof:

```
   <content>
         <module name="intellij.platform.settings.local"/>
   </content>
```

Reply:
Yes, I picked this up from the existing code. As you mentioned it fails without it. Your suggestions don't compile for me either.

Reply:
I am not sure I follow the reasoning in your comment, because (unlike IntelliJ) Sherlock doesn't include `IdeaPlugin.xml`, it only includes `JavaIdePlugin.xml`, so there shouldn't be any conflicts. 

The following builds successfully on my gLinux and Mac:

```
$ cat community-resources/resources/META-INF/SherlockPlatformPlugin.xml
<idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude">
  <!-- TODO: can the Java plugin be excluded somehow? -->
  <xi:include href="/META-INF/JavaIdePlugin.xml" xpointer="xpointer(/idea-plugin/*)"/>
  <xi:include href="/META-INF/essential-modules.xml"/>
</idea-plugin>
```

Reply:
It finally worked. Apparently IntelliJ was also caching it somehow - had to delete `.idea` along with `out/` before re-running the build.